pkg/registry/namespace/etcd/etcd_test.go

```diff
@@ -145,7 +145,7 @@ func TestDeleteNamespaceWithIncompleteFinalizers(t *testing.T) {
 		Spec: api.NamespaceSpec{
 			Finalizers: []api.FinalizerName{api.FinalizerKubernetes},
 		},
-		Status: api.NamespaceStatus{Phase: api.NamespaceActive},
+		Status: api.NamespaceStatus{Phase: api.NamespaceTerminating},
```

Comment:
Yes, related to https://github.com/kubernetes/kubernetes/pull/24964/files/87eef9bb78b897922842cd6ca39516e0ae5486d0#r61573647

It was testing against a namespace state that doesn't match what the rest storage ensures (deletionTimestamp and terminating phase get set together)